client/src/db/Browse.js

```diff
@@ -0,0 +1,528 @@
+import React from 'react';
+
+export const browse_platform = [
+    {
+        id: "1",
+        platform: "Playstation 5",
+        onclick: "clicky",
```

Comment:
remove all onClicks.  Our data is going to come from the database we won't be storing the function in an array.